e2e/generatorForm.playwright.ts

```diff
@@ -114,15 +114,15 @@ test.describe("Generator page functionality", () => {
     await fillEssentialFieldsWithDefaults(page);
 
     // Add two redirect URIs, delete one.
-    await page.locator(".redirectUris .AddNewButton").click();
+    await page.locator("#redirectUris .AddNewButton").click();
```

Comment:
It may be better to use `data-test-id`s for these instead of selectors

Reply:
I'm not sure, how that would work in those cases. I'll resolve it for now and take a note for the rest to be able to get merged :)